### DISCUSSION TITLE
Plugin decency on datepicker

### DISCUSSION THREAD

**Discussion by kookma** [NicolasPetton/Projectify#5] [General]
I am not sure how datepicker is used in Projectify but if it is only to choose due date why not use the standard html5 input tag.
As an example see my Shiraz plugin, Task Manager: https://kookma.github.io/TW-Shiraz/#demo%2Fdynamic-table%2Ftask-manager

Go to bottom of the page, click on the edit button left top of Task manager table and see how due date field works using standard html5 input tag.

Other reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

**Comment by NicolasPetton** [comment]
An input of type date would not work here, or I would have to do some tricks and hide it, as I want the trigger element to be a button.
I will get rid of the dependency anyway, as I plan to completely redo the date picker.
